src/app/containers/Timestamp/index.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { number } from 'prop-types';
 import moment from 'moment-timezone';
-import Timestamp from '../../components/Timestamp';
+import Timestamp from '@bbc/psammead-timestamp';
```

Comment:
Hey, can you pass the script into Timestamp please? Just realised it defaults to [latin](https://github.com/bbc/psammead/blob/latest/packages/components/psammead-timestamp/src/index.jsx#L35) which we shouldn't have done 🙈 We should pass latin in explicitly, make `script` required and remove the default

Reply:
Ive made https://github.com/bbc/psammead/issues/481 to remove the default :) doesnt need to be done before this PR, but we should anticipate it by passing in the script imo :)

Reply:
I think we should maybe wrap it in `ServiceContextConsumer` so that we can use the appropriate `script` based on the context just as we do in `Brand` for the `brandSVG`.

Reply:
We can use the `useContext` hook, but yeah sounds good :)